src/util/datetime.ts

```diff
@@ -0,0 +1,9 @@
+import { DateTime } from 'luxon';
+
+export function formatDatetime(date: Date): string {
+  const parsedTime: DateTime = DateTime.fromJSDate(date);
+  if (!parsedTime.isValid) return undefined;
+  const front: string = parsedTime.toFormat('EEE, MMM dd ');
+  const back: string = parsedTime.toFormat('yy hh:mm a');
+  return front + "'" + back;
```

Comment:
is there any chance parsedTime.toFormat() to return error? what would the returned result be?

Reply:
no, as long as it's a valid luxon datetime cmiiw